src/views/empty-state.html

```diff
@@ -2,13 +2,13 @@
   <div ng-if="$ctrl.config.icon" class="blank-slate-pf-icon">
     <span class="{{$ctrl.config.icon}}"></span>
   </div>
-  <h1 id="title">
+  <h4 id="empty-state-title-{{$id}}" class="h1 title">
     {{$ctrl.config.title}}
-  </h1>
-  <p id="info" ng-if="$ctrl.config.info">
+  </h4>
+  <p id="empty-state-info-{{$id}}" class="info" ng-if="$ctrl.config.info">
```

Comment:
I can update this in the tests, though want to bring up that someone targeting with `css` will have a lousy time trying to get at these without a class hook at all (and such a specific id as `empty-state-info-54`.  Are we sure we don't want a hook at all?  (I tend to come from the camp that believes you should never write `css` targeting dom nodes).